src/AssemblerOptions.cpp

```diff
@@ -569,6 +574,7 @@ void AssemblerOptions::addConfigurableOptions()
 
 void AssemblerOptions::ReadsOptions::PalindromicReadOptions::write(ostream& s) const
 {
+    s << "palindromicReads.skipFlagging = " << skipFlagging << "\n";
```

Comment:
Need to use `convertBoolToPythonString` as done for other Boolean switches. This is necessary so the configuration file stored in the assembly directory can be used by the Python API.

Reply:
Ah. Didn't realize that. Will do.